### ISSUE TITLE
Possibility of absolute timing

### ISSUE THREAD

**Issue by ArthurClemens** [sergi/jsmidi#1]
Currently the event timings are relative, making it hard (impossible?) to use my absolute timings. The author of Perl MIDI has made a case of using absolute timings for scores (see http://search.cpan.org/~sburke/MIDI-Perl-0.81/lib/MIDI/Score.pm). Would be great if jsmidi would also support something like this.

**Comment by sergi** [comment]
Yeah, I agree this would be a nice thing to have. I'll think about putting it in.

Thanks for the suggestion :)

**Comment by rdroe** [comment]
I know this is an older thread (and repo), but I second this suggestion. 

Thanks for your work, Sergi; I've gotten some really fun use out of jsmidi!

I started using jsmidi because it's api is simpler than the alternatives I've found, which is really useful.
